VelocityPacketTrackerFirstObjective.ipynb

```diff
@@ -0,0 +1,3493 @@
+{
```

Comment:
<p>I don't think it's meaningful to show legend in this plot since we have no other color</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2538/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Removed.